src/consensus/validation.h

```diff
@@ -74,16 +74,16 @@ class CValidationState {
         MODE_INVALID, //!< network rule violation (DoS value may be set)
         MODE_ERROR,   //!< run-time error
     } mode;
-    ValidationInvalidReason reason;
+    ValidationInvalidReason m_reason;
```

Comment:
Can you squash "nit: reason -> m_reason"? Please don't add full commits that just address nits. Ignoring the nit is also perfectly valid.

Reply:
Fixed.